src/commonMain/kotlin/exchange.dydx.abacus/calculator/TriggerOrdersInputCalculator.kt

```diff
@@ -89,10 +90,10 @@ internal class TriggerOrdersInputCalculator(val parser: ParserProtocol) {
         val positionSide = parser.asString(parser.value(position, "resources.indicator.current"))
         val positionSize = parser.asDouble(parser.value(position, "size.current"))?.abs() ?: return modified
         val notionalTotal = parser.asDouble(parser.value(position, "notionalTotal.current")) ?: return modified
-        val leverage = parser.asDouble(parser.value(position, "leverage.current"))?.abs() ?: return modified
+        val leverage = max(parser.asDouble(parser.value(position, "leverage.current"))!!.abs(), 1.0)
```

Comment:
nit: prefer `requireNotNull()` over `!!` since the former allows for an error message to be set.